odoo/cli/i18n.py

```diff
@@ -0,0 +1,164 @@
+        if not db_names or len(db_names) > 1:
+            Command.die("Please provide a single database in the config file.")
+        db_name = db_names[0]
+
+        subcommand_handler = {
+            'import': self._import,
+            'export': self._export,
+            'loadlang': self._loadlang,
+        }[parsed_args.subcommand]
+
+        # Run subcommand
+        registry = Registry.new(db_name)
+        with closing(registry.cursor()) as cr:
+            env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
+            subcommand_handler(env, parsed_args)
+
+    def _import(self, env, parsed_args):
+        if parsed_args.format in ('tgz', 'csv') and not parsed_args.lang:
+            Command.die("Language must be specified when '*.tgz' and '*.csv' files are imported.")
+        if parsed_args.format == 'csv':
+            # The default limit for CSV fields in the module is 128KiB, which is not
+            # quite sufficient to import images to store in attachment. 500MiB is a
+            # bit overkill, but better safe than sorry
+            csv.field_size_limit(500 << 20)
+
+        translation_importer = odoo.tools.translate.TranslationImporter(env.cr)
+        for filename in parsed_args.files:
+            if parsed_args.format == 'pot':
+                lang = parsed_args.lang
+            elif Path(filename).suffixes[-1:] == ['.po']:
+                code = Path(filename).stem
+                domain = ['|', ('code', '=', code), ('code', 'ilike', f'{code}%')]
+                lang = env['res.lang'].search(domain, limit=1).code
+            translation_importer.load_file(filename, lang)
+        translation_importer.save(overwrite=not parsed_args.only_new)
+
+    def _export(self, env, parsed_args, logging_enabled=True):
+
+        # When outputting to stdout, disable the logging
+        if parsed_args.to_stdout and logging_enabled:
+            logging.disable(logging.ERROR)
+            try:
+                self._export(env, parsed_args, logging_enabled=False)
+            finally:
+                logging.disable(logging.NOTSET)
+            return
+
+        if export_pot := 'pot' in parsed_args.lang:
+            pot_idx = parsed_args.lang.index('pot')
+            parsed_args.lang.pop(pot_idx)
+
+        # Get language files
+        language_codes = self._get_languages(env, parsed_args.lang)
+        language_files = []
+        codes = [[None, 'pot'] if export_pot else []] + (list(language_codes) if language_codes else [])
+        modules_map = {m.name: m for m in env['ir.module.module'].search([('name', 'in', parsed_args.modules)])}
+        for module_name, (code, isocode) in product(parsed_args.modules, codes):
+            if module := modules_map.get(module_name):
+                filename = f"{module_name}.pot" if isocode == 'pot' else f"{isocode}.po"
+                filepath = str(Path(get_module_path(module_name)) / 'i18n' / filename)
+                language_files.append([module, code, filepath])
```

Comment:
actually not a nitpicking, a tuple is indeed the indicated datastructure to use in this case *and* is much faster to instantiate than a list.
```suggestion
                language_files.append((module, code, filepath))
```